fs-one-concepts.md

```diff
@@ -3,19 +3,51 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
+Enzyme is a testing library for React applications that is often used within the Jest framework for unit
+testing. It allows you to test the DOM with mount by creating a testing instance of the class and using simulate
+to recreate user behavior.
 
 #### 2. What is the difference between dynamic and a static routes?
+Static routes only work for that specific route whereas a dynamic route is able to change its parameters so 
+it's much more flexible.
 
 #### 3. What is a JSON API?
+JavaScript Object Notation Application Programming Interface: It's a software that allows 2 applications to 
+communicate with each other and transfer JSON objects easily.
 
 #### 4. What is a migration and why would you use one?
+A migration is a command in Rails that allows you to modify the database to include more columns or delete
+a field. It provides a convenient history you can check.
 
 #### 5. Explain how to set up a route in React.
+Each React application will need route directions in route.db
```

Comment:
You giving a great example of how to set up Rails routes. What are the steps to setting up routes in React?